src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,54 @@
+import classnames from 'classnames';
+import { noOp, px } from '../../utils';
+import CloseIcon from '../../assets/close.svg?react';
+import Button from '../Button/Button';
+import ReactModal from 'react-modal';
+
+export interface ModalProps extends ReactModal.Props {
+  /**
+   * Function to close the modal
+   */
+  onClose?: () => void;
+}
```

Comment:
Do we want to describe the `isOpen` prop in our interface here for documentation?